src/profiler/profiler.h

```diff
@@ -807,6 +810,13 @@ struct ProfileTask : public ProfileDuration {
 
   ProfileObjectType type() const override { return kTask; }
 
+  /*!
+   * \brief Whether to add stat to AggregateStats
+   */
+  void enableAggregateStats(bool enabled) {
```

Comment:
In the all the places this is True. Except in VTune task case.
Can we make it default parameter with default value True?
So in only VTune Task case, we set it to False. In other places, we don't need to make changes.

Reply:
I can default enabled too true. enableAggregateStats is a new function I added and the only place it is used is in line 1167

Reply:
enable_aggregate_ has already been defaulted to true